<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDlg</class>
 <widget class="QDialog" name="SettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabConnection">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>168</width>
            <height>235</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Connection Status: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelLight"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>User name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leUserName"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Set Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Server address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leAddress"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lePort"/>
           </item>
           <item>
            <widget class="QLabel" name="labelMessage"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabClient">
      <attribute name="title">
       <string>Client</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>168</width>
            <height>162</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="labelSubtlety">
             <property name="text">
              <string>Animation subtlety</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderSubtlety">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelFontSize">
             <property name="text">
              <string>Font size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderFontSize">
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>16</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbLightTrail">
             <property name="text">
              <string>Show light trail</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbAfterImage">
             <property name="text">
              <string>Show afterimage</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
